spec/forms/generic_file_edit_form_spec.rb

```diff
@@ -1,11 +1,9 @@
 require 'spec_helper'
 
 describe Hydranorth::Forms::GenericFileEditForm do
-  subject { described_class.new(GenericFile.new) }
-
   describe "#terms" do
     it "should return a list" do
-      expect(subject.terms).to eq([:title, :creator, :contributor, :subject, :resource_type, :language, :spatial, :temporal, :description, :date_created, :license, :rights, :is_version_of, :source, :related_url, :belongsToCommunity, :hasCollectionId ])
+      expect(Hydranorth::Forms::GenericFileEditForm.terms).to eq([:title, :creator, :contributor, :subject, :resource_type, :language, :spatial, :temporal, :description, :date_created, :doi, :license, :rights, :is_version_of, :source, :related_url, :belongsToCommunity, :hasCollectionId ])
```

Comment:
Highly against subject syntax, especially when its confusing magic like this one.

```subject { described_class.new(GenericFile.new) }
subject.terms
isn't intuitive on what is going on.
```